app/views/system_admin/providers/show.html.erb

```diff
@@ -1 +1,29 @@
 <%= render PageTitle::View.new(title: @provider.name, has_errors: @provider.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
```

Comment:
I think this backlink should be directly above the `<h1>`, otherwise the layout jumps around when you click through. Like this:
<img width="832" alt="Screenshot 2021-02-03 at 16 14 22" src="https://user-images.githubusercontent.com/18436946/106775496-14066880-663b-11eb-9ca3-f00ca51f26d5.png">

Reply:
Ok I've added a backlink to the new view and then moved them above the h1 to keep the system admin panel static. As there is a margin built into the backlink component its not quite level with the system admin h2, not sure if an issue.
![image](https://user-images.githubusercontent.com/58793682/106789085-b4638980-6649-11eb-8e0e-7bce91885f43.png)

Reply:
Nice! Happy with the margin thing - i think system admins can deal 👍